src/main/java/com/samoilenko/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -39,6 +39,14 @@ protected ResponseEntity<Object> handleEntityNotFoundException(
         return getResponseEntity(HttpStatus.NOT_FOUND, bodyOfResponse);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    protected ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex, WebRequest request
+    ) {
+        String bodyOfResponse = "Can't register user" + ex.getMessage();
+        return getResponseEntity(HttpStatus.CONFLICT, bodyOfResponse);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

> The HTTP 409 Conflict response status code indicates a request conflict with the current state of the target resource.
Conflicts are most likely to occur in response to a [PUT](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT) request. For example, you may get a 409 response when uploading a file that is older than the existing one on the server, resulting in a version control conflict.

Registration is about the POST request and creating the entity, to me this looks more like a usual `400 - bad request`